tests/lib/snaps/snap-install-hooks-broken1/bin/app

```diff
@@ -0,0 +1 @@
+#!/bin/sh
```

Comment:
If the command is not going to be used we could declare it as `command: "true"` and remove `bin/app`